test/models/paymentChannelCreate.ts

```diff
@@ -0,0 +1,210 @@
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": "10000",
+            "SettleDelay": 86400,
+            "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A",
+        } as any       
+        
+        assert.throws(
+            () => verifyPaymentChannelCreate(missingDest),
+            ValidationError,
+            "PaymentChannelCreate: missing Destination"
+        )
+    })
+
+    it (`missing SettleDelay`, () => {
+        const missingSettleDelay = {
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": "10000",
+            "Destination": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A",
+        } as any       
+        
+        assert.throws(
+            () => verifyPaymentChannelCreate(missingSettleDelay),
+            ValidationError,
+            "PaymentChannelCreate: missing SettleDelay"
+        )
+    })
+
+    it (`missing PublicKey`, () => {
+        const missingKey = {
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": "10000",
+            "Destination": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            "SettleDelay": 86400
+        } as any       
+        
+        assert.throws(
+            () => verifyPaymentChannelCreate(missingKey),
+            ValidationError,
+            "PaymentChannelCreate: missing PublicKey"
+        )
+    })
+
+    it (`invalid Amount`, () => {
+        const missingAmt = {
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": 10,
+            "Destination": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            "SettleDelay": 86400,
+            "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A",
+        } as any
+        
+        assert.throws(
+            () => verifyPaymentChannelCreate(missingAmt),
+            ValidationError,
+            "PaymentChannelCreate: invalid Amount"
+        )
+    })
+
+
+    it (`invalid Destination`, () => {
+        const missingDest = {
+            "Account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            "TransactionType": "PaymentChannelCreate",
+            "Amount": "10000",
+            "Destination": 10,
+            "SettleDelay": 86400,
+            "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A",
+        } as any       
+        
+        assert.throws(
+            () => verifyPaymentChannelCreate(missingDest),
+            ValidationError,
+            "PaymentChannelCreate: invalid Destination"
+        )
+    })
+
+    it (`invalid SettleDelay`, () => {
+        const missingSettleDelay = {
```

Comment:
nit: This should probably be called invalidSettleDelay instead of missingSettleDelay